gerritlab/main.py

```diff
@@ -126,92 +121,78 @@ def cancel_prev_pipelines(commit):
                 local_branch, utils.get_change_id(commits[idx - 1].message))
         commits_data.append(Commit(c, source_branch, target_branch))
 
-    # Before we update the existing the MRs or create new MRs, there are a few
```

Comment:
Good to see this go away! This was probably the most confusing part in GerritLab.

Reply:
Agreed :100: